packages/python/plotly/js/widget.ts

```diff
@@ -1,7 +1,7 @@
 // Adapted from ../../../javascript/jupyterlab-plotly/src/Figure.ts for anywidget.
-import type { DOMWidgetModel } from "@jupyter-widgets/base";
-import _ from "lodash";
+import _ from "https://esm.sh/lodash-es@4.17.21";
```

Comment:
Adding an esm URL like this will break the build in enviroments that don't have an internet connection (which you probably don't want). I'd maybe first try `npm install lodash-es` and importing from `lodash-es` here and see if it works.

Reply:
It does work! Thank you, I'll push that